grr/server/grr_response_server/gui/ui/components/client_search/module.ts

```diff
@@ -5,6 +5,7 @@ import {RouterModule} from '@angular/router';
 
 import {ClientSearch} from './client_search';
 import {ClientSearchRoutingModule} from './routing';
+import { TimestampModule } from '../timestamp/module';
```

Comment:
Does your formatter format with spaces around import literals `{` and `}`? If so, let's chat about formatter configuration.

Reply:
This was added before updating the formatter, that's why it has the spaces.